features/0509-action-menu/README.md

```diff
@@ -0,0 +1,211 @@
+# Aries RFC 0509: Action Menu Protocol 0.9
+
+- Authors: [Andrew Whitehead],[Stephen Curran]
+- Status: [PROPOSED](/README.md#proposed)
+- Since: 2019-12-26 (date you submit your PR)
+- Status Note: (explanation of current status)  
+- Supersedes: 
+- Start Date: 2020-06-24 (date you started working on this idea)
+- Tags: [feature](/tags.md#feature)
+
+## Summary
+
+The action-menu protocol allows one Agent to present a set of heirarchical menus and actions to another user-facing Agent in a human friendly way. The protocol allows limited service discovery as well as simple data entry. While less flexible than HTML forms or a chat bot, it should be relatively easy to implement and provides a user interface which can be adapted for various platforms, including mobile agents.
+
+## Motivation
+
+Discovery of a peer Agent's capabilities or service offerings is currently reliant on knowledge obtained out-of-band.  There is no in-band DIDComm supported protocol for querying a peer to obtain a human freindly menu of their capabilities or service offerings. Whilst this protocol doesn't offer ledger wide discovery capabilities, it will allow one User Agent connected to another, to present a navigable menu and request offered services. The protocol also provides an interface definition language to define action menu display, selection and request submission.
+
+## Tutorial
+
```

Comment:
Should add the protocol name and version here:

```
### Name and Version

`present-proof`, version 2.0
```

Reply:
Done